app/feature/feature-travel-certificate/src/main/kotlin/com/hedvig/android/feature/travelcertificate/ui/history/CertificateHistoryViewModel.kt

```diff
@@ -137,12 +147,16 @@ private sealed interface ScreenContentState {
 
   data object Loading : ScreenContentState
 
-  data class Success(val certificateHistoryList: List<TravelCertificate>, val eligibleToCreateCertificate: Boolean) :
+  data class Success(
+    val certificateHistoryList: List<TravelCertificate>,
+    val eligibleToCreateCertificate: Boolean,
+    val hasChooseOption: Boolean,
```

Comment:
Could we name this a bit more descriptive? If they have a choice to choose something if you are lacking the context of what that choice is does not tell the entire story.
Something like `mustChooseContractBeforeGeneratingTravelCertificate` is the first thing that pops to my mind, to kinda steer you more towards what I would like this to say to be super clear about its purpose.